packages/components/src/Icon/_docs/icon.stories.scss

```diff
@@ -12,15 +12,12 @@
 
 .button {
   @include button-reset;
-
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  gap: var(--spacing-16);
-  padding: var(--spacing-8);
+  display: grid;
+  grid-template-rows: min-content 1fr;
```

Comment:
Oops my bad! Didn't see that there was actually a `.grid` controlling the grid - thought that `.button` was the grid.
So technically it does make sense for this to be display flex, but the height not being a set value is still valid.

Nonetheless, up to you if you want to change it or not - visually it looks much better than before already (and it's internal docs, so I'm not so fussed) :)

Reply:
All breezy, I've revert part of this back and leveraged the upper grid to control the width of the columns